test/unittests/frontend.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/*
+ * This is a unit test suite that makes sure various parts of the app function
+ * as expected
+ */
+
+const assert = require('assert');
+var requirejs = require("requirejs");
+requirejs.config({
+  baseUrl: '../sources/web/datalab/static/',
+  nodeRequire: require,
+  map: {
+    '*': {
+      util: 'utilMock'
+    }
+  }
+});
+
+describe('Unit tests', function() {
```

Comment:
No confusion, it's just organizational. We now have three suites of tests, each starts off with its category name, UI, notebook, or unit tests, then the name of the test.